lib/core/src/display/shape/text/text_field/content.rs

```diff
@@ -244,6 +277,13 @@ impl TextFieldContent {
         }
     }
 
+    /// Apply many changes to content.
+    pub fn make_changes<Changes:IntoIterator<Item=TextChange>>(&mut self, changes:Changes) {
```

Comment:
As docs tell, `apply_changes` would be better than `make_changes`. Of course this is just a wording here.